sites/public/src/layouts/application.tsx

```diff
@@ -21,12 +20,83 @@ const Layout = (props) => {
       </div>
 
       <SiteFooter>
-        <FooterNav copyright={t("footer.copyright")}>
-          <Link href="/privacy">{t("pageTitle.privacy")}</Link>
-          <Link href="/disclaimer">{t("pageTitle.disclaimer")}</Link>
-        </FooterNav>
-        <FooterSection className="bg-black" small>
-          <ExygyFooter />
+        <FooterSection sectionClassName="justify-around" small>
```

Comment:
this is a lot of implementation detail code in application.tsx especially since this component will be reused (e.g. partners portal) + any other layout we need to write or anywhere else that we need to put a footer

Could we find a way to make this one step better without necessarily making you rewrite everything given the time crunch here? 
Some options include:
* dumping it into a helper function like #180
* making a doorway footer component that basically just has all of this in it

Reply:
Yeah sorry that was messy, used a helper function. Thanks for the suggestion